frontend/src/concepts/pipelines/apiHooks/usePipelines.ts

```diff
@@ -43,4 +49,44 @@ export const useSafePipelines = (
   );
 };
 
+async function getAllPipelines(
+  opts: K8sAPIOptions,
+  params: PipelineParams | undefined,
+  listPipelines: ListPipelines,
+): Promise<PipelineKFv2[] | undefined> {
```

Comment:
Would this be better to be `PipelineKFv2[]` type? We can always set `undefined` as an empty array. It's easier for an array type.

Reply:
I don't have much of an opinion on this. I don't think it improves or changes much, but I'll just update it.